src/Driver/driver_configs.jl

```diff
@@ -583,9 +583,12 @@ function SingleStackConfiguration(
     numerical_flux_second_order = CentralNumericalFluxSecondOrder(),
     numerical_flux_gradient = CentralNumericalFluxGradient(),
     fv_reconstruction = nothing,
+    Ncutoff = get_polyorders(N),
```

Comment:
I suppose `Ncutoff` is defined so we can set the cutoff manually, but can't we do that already through `ClimateMachine.Settings.cutoff_degree`?

Reply:
I kind of just followed the existing pattern from the LES/GCM configs 🤷🏻‍♂️

Reply:
That's fine then. If anything, it's redundant but it doesn't harm